.github/workflows/docs-ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Docs CI
+on:
+  push:
+    branches: [main]
+    paths:
+    - 'docs/**'
+    - '!docs/**/*.md'
+  pull_request:
+    branches: [main]
+    paths:
+    - 'docs/**'
+    - '!docs/**/*.md'
+
+jobs:
+  docs_ci:
+    name: Verify Files
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+    - name: Install Docs Packages
+      working-directory: docs
+      run: npm install
+    - name: Stylelint Docs
+      working-directory: docs
+      run: npm run lint:scss
+    - name: Install Packages
+      run: npm install
```

Comment:
why are we installing deps twice?

Reply:
```suggestion
```

Reply:
Please note that the installation depends on two different directories.

Because stylelint is in the `/docs` directory and eslint is in the `/` directory, you need to install it twice to execute the check properly.